### DISCUSSION TITLE
Filter events in widget

### DISCUSSION THREAD

**Discussion by cpoisnel** [pretix/pretix#2393] [Ideas]
Hi,

We use the widget to show organizer events (all public events, as mentioned in doc : https://docs.pretix.eu/en/latest/user/events/widget.html#multi-event-selection), but it seems that it is not possible to filter from this view by 
 event ids. 

Would it be possible to filter by events?
Regards,

**Comment by raphaelm** [comment]
It should be possible to filter by meta data, IIRC. What widget code are you trying exactly?
